### PR TITLE
Docs/usman plugin mgmt

### PR DESCRIPTION
**Pull request by usmangt**
This PR fixes some of the missing information and adds more clarity for the page:

https://grafana.com/docs/grafana/latest/administration/plugin-management/

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/administration/plugin-management/index.md [hunk 1]
```diff
@@ -17,9 +17,13 @@ weight: 600
 
 Besides the wide range of visualizations and data sources that are available immediately after you install Grafana, you can extend your Grafana experience with _plugins_.
 
-You can [install](#install-a-plugin) one of the plugins built by the Grafana community, or [build one yourself](/developers/plugin-tools).
+You can [install](#install-a-plugin) a wide range of the plugins that are built by the Grafana Labs, Partners, Community, or [build one yourself](/developers/plugin-tools).
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:20]:
```suggestion
You can [install](#install-a-plugin) a wide range of the plugins that are built by Grafana Labs, commercial partners, our community, or [build one yourself](/developers/plugin-tools).
```

File: docs/sources/administration/plugin-management/index.md [hunk 2]
```diff
@@ -17,9 +17,13 @@ weight: 600
...
 
-Grafana supports three types of plugins: [panels](/grafana/plugins?type=panel), [data sources](/plugins?type=datasource), and [apps](/grafana/plugins?type=app).
+Grafana supports three types of plugins:
+
+- [Panels](/grafana/plugins?type=panel)
+- [Data sources](/plugins?type=datasource)
+- [Apps](/grafana/plugins?type=app)
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:26]:
```suggestion
- [Panels](/grafana/plugins/panel-plugins)
- [Data sources](/grafana/plugins/data-source-plugins)
- [Apps](/grafana/plugins/app-plugins)
```

Its odd that they were different, and also redirects so maybe better to go with where they actually land - (please verify this with a build of the docs)

File: docs/sources/administration/plugin-management/index.md [hunk 3]
```diff
@@ -33,19 +37,19 @@ Use panel plugins when you want to:
 
 ## Data source plugins
 
-Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource).
+Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource), [Zendesk](https://grafana.com/grafana/plugins/gabrielthomasjacobs-zendesk-datasource/) and [NetData](https://grafana.com/grafana/plugins/netdatacloud-netdata-datasource/) Data source plugins.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:40]: Required change: Zendesk and NetData aren't what I would think of as databases, and bookending the start and end with "Data source plugins" feels strange to me.

File: docs/sources/administration/plugin-management/index.md [hunk 4]
```diff
@@ -33,19 +37,19 @@ Use panel plugins when you want to:
...
 
 Data source plugins communicate with external sources of data and return the data in a format that Grafana understands. By adding a data source plugin, you can immediately use the data in any of your existing dashboards.
 
-Use data source plugins when you want to import data from external systems.
+Use data source plugins when you want to import data from external or 3rd party systems.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:44]: This is a slightly strange sentence given data source plugins are also how we query things like loki whether thats in the same Grafana stack in cloud or self hosted.

File: docs/sources/administration/plugin-management/index.md [hunk 5]
```diff
@@ -33,19 +37,19 @@ Use panel plugins when you want to:
...
 
 ## App plugins
 
-Applications, or _app plugins_, bundle data sources and panels to provide a cohesive experience, such as the [Zabbix](/grafana/plugins/alexanderzobnin-zabbix-app) app.
+Applications, or _app plugins_, bundle data sources and panels to provide a cohesive experience, such as the [Zabbix](/grafana/plugins/alexanderzobnin-zabbix-app), [Performance Co-Pilot](https://grafana.com/grafana/plugins/performancecopilot-pcp-app/) and [cPacket Utility Bundle](https://grafana.com/grafana/plugins/cpacket-cclearutilitybundle-app/) apps.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:48]: This is generally a bad description of apps given theres no requirement to bundle anything... which I know is outside the scope of these changes, BUT technically, the cPacket one doesn't include a data source (the others do) so unless we change the description, we should change the example.

File: docs/sources/administration/plugin-management/index.md [hunk 6]
```diff
@@ -61,13 +65,11 @@ To prevent users from seeing an app plugin, refer to [this permissions scenarios
...
 
-| Org Admin | Server Admin | Permissions                                                                                 |
-| --------- | ------------ | ------------------------------------------------------------------------------------------- |
-| &check;   | &check;      | <ul><li>Can configure app plugins</li><li>Can install/uninstall/update plugins</li></ul>    |
-| &check;   | &times;      | <ul><li>Can configure app plugins</li><li>Cannot install/uninstall/update plugins</li></ul> |
-| &times;   | &check;      | <ul><li>Cannot configure app plugins</li><li>Can install/uninstall/update plugins</li></ul> |
+- If you are an **Org Admin**, you can configure app plugins but you cannot install/uninstall/update them.
+- If you are a **Server Admin**, you cannot configure app plugins but you can install/uninstall/update them.
+- If you are both (**Org + Server Admin**), you can configure app plugins and also install/uninstall/update them.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:72]: I still feel like this is confusing, but this time because it implies that it is only about app plugins, whereas install/uninstall/update refers to **all** plugins. 

@josmperez perhaps you could suggest something?

I had originally suggested:
> - The **server admin** role is required to install/uninstall/update plugins.
> - The **org admin** role is required to configure app plugins.

Maybe a change to the preceding paragraph would also help

File: docs/sources/administration/plugin-management/index.md [hunk 7]
```diff
@@ -81,6 +83,11 @@ The Plugin catalog allows you to browse and manage plugins from within Grafana.
 _Video shows the Plugin catalog in a previous version of Grafana._
 
 In order to be able to install / uninstall / update plugins using plugin catalog, you must enable it via the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
+
+{{% admonition type="note" %}}
+In the latest versions of Grafana, the `plugin_admin_enabled` flag is enabled by default.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:88]: I'd be tempted to just remove this and the previous paragraph instead.

File: docs/sources/administration/plugin-management/index.md [hunk 8]
```diff
@@ -129,7 +136,11 @@ When the update is complete, you see a confirmation message that the uninstall w
 
 ## Install Grafana plugins
 
-Grafana supports data source, panel, and app plugins. Having panels as plugins makes it easy to create and add any kind of panel, to show your data, or improve your favorite dashboards. Apps enable the bundling of data sources, panels, dashboards, and Grafana pages into a cohesive experience.
+Grafana supports data source, panel, and app plugins.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:139]: This whole paragraph feels like unnecessary repetition of content we've covered above, or should have. At this stage it is task based documentation and we shouldn't bloat it.

File: docs/sources/administration/plugin-management/index.md [hunk 9]
```diff
@@ -144,11 +155,13 @@ If you are logged in to Grafana Cloud when you add a plugin, log out and back in
 
 ### Install plugin on local Grafana
 
-Follow the instructions on the Install tab. You can either install the plugin with a Grafana CLI command or by downloading and uncompress a .zip file into the Grafana plugins directory. We recommend using Grafana CLI in most instances. The .zip option is available if your Grafana server does not have access to the internet.
+Follow the instructions on the Install tab. You can either install the plugin with a Grafana CLI command or by downloading and uncompress a `.zip` file into the Grafana plugins directory. We recommend using Grafana CLI in most instances. The `.zip` option is available if your Grafana server does not have access to the internet.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:158]: The challenge I have is that we have just described the primary installation method, the catalog. 

As minor editorial changes, these are fine, but I feel they are not improving the page - as we should be thinking about whether the sections are required or should be reframed.

For Cloud, given the default is now that plugin installation is the same as on premise, and we have Grafana Cloud docs, this section feels pointless within this page and is more a "detail" for the cloud docs perhaps. The for local, instead this could be about "Installing plugins using the Grafana CLI"

File: docs/sources/shared/find-and-use-plugins.md
Comment (sympatheticmoose) [inline docs/sources/shared/find-and-use-plugins.md:1]: As I don't have a local build running, just want to check - is this file intended to be in this PR and how is it intended to show? I'm not sure where the fundamentals/how-do-i are meant to then show up

File: docs/sources/administration/plugin-management/index.md (hunk 3 above)
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md:40]: The reason I mentioned them is that I want to share the combination of Data sources which are owned by Grafana (Zabbix), Community (Zendesk) and Partner (Netdata). 

If you suggest that it may not be the required then I can simply undo my changes here.

File: docs/sources/administration/plugin-management/index.md (hunk 4 above)
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md:44]: I actually took it from https://grafana.com/developers/plugin-tools/introduction/plugin-types-usage#usage-of-data-source-plugins where it says "when you want to work with data provided by a third-party service".

File: docs/sources/administration/plugin-management/index.md (hunk 5 above)
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md:48]: Same intention as mentioned above in Data Sources. 

Since I am not too expert with Apps so I can simply undo the changes here.

File: docs/sources/administration/plugin-management/index.md (hunk 7 above)
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md:88]: Acknowledge

File: docs/sources/administration/plugin-management/index.md (hunk 8 above)
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md:139]: I also agree here as this felt repetition for me too and I also agree that we should simply remove it

File: docs/sources/administration/plugin-management/index.md (hunk 9 above)
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md:158]: Good Point.

I suggest the following. We keep the heading https://grafana.com/docs/grafana/latest/administration/plugin-management/#install-plugin-on-local-grafana

And then mention that users can use the Plugin Catalog as it is available by default. And if want to install via CLI then refer them to the https://grafana.com/docs/grafana/latest/cli/ which itself is very detailed (I can move the unzip example there so that we do not loose it).

For the cloud do you know the page which we can refer to?

File: docs/sources/shared/find-and-use-plugins.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+1. In Grafana's right-hand menu, click **Administration** > **Plugins and data** > **Plugins** to view installed plugins.
+1. Click the **All** filter to browse all available plugins.
+1. Click the **Data sources**, **Panels**, or **Applications** buttons to filter by plugin type.
+1. Click the logo of the plugin you want to install.
+1. Click install.
+
+## Additional information
+
+For more information on Grafana plugins, refer to the [documentation](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
```
Comment (jdbaldry) [inline docs/sources/shared/find-and-use-plugins.md:43]: The link text should be more description. Perhaps:

```suggestion
For more information on Grafana plugins, refer to [Plugin management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
```

File: docs/sources/shared/find-and-use-plugins.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+1. Optional: select the desired version. We strongly recommended to run the latest version of plugins.
+1. Click **Install plugin** on the appropriate Grafana instance.
+
+### Install a plugin from Grafana
+
+1. In Grafana's right-hand menu, click **Administration** > **Plugins and data** > **Plugins** to view installed plugins.
+1. Click the **All** filter to browse all available plugins.
+1. Click the **Data sources**, **Panels**, or **Applications** buttons to filter by plugin type.
+1. Click the logo of the plugin you want to install.
+1. Click install.
```
Comment (jdbaldry) [inline docs/sources/shared/find-and-use-plugins.md:39]: If this is a UI button, we should make sure that it's spelled the exact same way as in the UI and it should also be in bold https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold

File: docs/sources/shared/find-and-use-plugins.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+menuTitle: Find and use plugins
+title: Find and use plugins
+weight: 750
+---
+
+# Find and use Grafana plugins
+
+Plugins extend Grafana by adding capabilities that are not already present. There are plugins that add the ability to import data from diverse sources, that bundle data sources and panels, and that provide new visualization types for use in dashboards.
+
+To find new plugins, browse the Grafana plugins catalog which is available both [within Grafana](https://grafana.com/docs/grafana/latest/administration/plugin-management/#browse-plugins) and on the Grafana [website](/grafana/plugins/).
```
Comment (jdbaldry) [inline docs/sources/shared/find-and-use-plugins.md:24]: Links should use the special syntax for version substitution so that different versions of docs don't all link to "latest".

```suggestion
To find new plugins, browse the Grafana plugins catalog which is available both [within Grafana](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/plugin-management/#browse-plugins) and on the Grafana [website](/grafana/plugins/).
```

**Review by josmperez — COMMENTED** [review]
Great updates! I've completed a technical edit and have added my proposed changes in [this commit](https://github.com/grafana/grafana/pull/85424/commits/19c0827c4e563bd3ebc81a69884dedb9c9e6dcd5). Please review these changes to ensure that they haven't introduced any inaccuracies. I think the reasons for the changes should be evident, especially when compared to the style guidelines in the Writer's Toolkit. However, if you don't understand why a change has beenn suggested, just ask.

File: docs/sources/administration/plugin-management/index.md (hunk 8 above)
Comment (josmperez) [reply docs/sources/administration/plugin-management/index.md:141]: In my latest commit, I moved to the content up to the top where types were introduced without description, and removed it here. Does that improve?

File: docs/sources/administration/plugin-management/index.md [hunk 13]
```diff
@@ -177,26 +184,28 @@ If you are a plugin developer and want to know how to sign your plugin, refer to
 | Signature status   | Description                                                                     |
 | ------------------ | ------------------------------------------------------------------------------- |
 | Core               | Core plugin built into Grafana.                                                 |
-| Invalid signature  | The plugin has a invalid signature.                                             |
+| Invalid signature  | The plugin has an invalid signature.                                             |
 | Modified signature | The plugin has changed since it was signed. This may indicate malicious intent. |
 | Unsigned           | The plugin is not signed.                                                       |
 | Signed             | The plugin signature was successfully verified.                                 |
 
 ### Plugin signature levels
```
Comment (josmperez) [inline docs/sources/administration/plugin-management/index.md:192]: In the Developer Portal, we have removed a similar section and instead link directly to the [official definitions][(https://grafana.com/legal/plugins/#what-are-the-different-classifications-of-plugins](https://grafana.com/legal/plugins/#what-are-the-different-classifications-of-plugins)) provided by Grafana Legal. This eliminates inconsistencies and improves maintainability. Would that work here?

Comment (sympatheticmoose) [reply docs/sources/administration/plugin-management/index.md:192]: These are different, its more about the signature check than the signature type

File: docs/sources/administration/plugin-management/index.md [hunk 14]
```diff
@@ -80,91 +88,91 @@ The Plugin catalog allows you to browse and manage plugins from within Grafana.
 
 _Video shows the Plugin catalog in a previous version of Grafana._
 
-In order to be able to install / uninstall / update plugins using plugin catalog, you must enable it via the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
-Before following the steps below, make sure you are logged in as a Grafana administrator.
+In order to be able to install, uninstall, or update plugins using the plugin catalog, you must enable it with the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
 
-<a id="#plugin-catalog-entry"></a>
+{{% admonition type="note" %}}
+In the most recent versions of Grafana, the `plugin_admin_enabled` flag is enabled by default.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:94]:
```suggestion
If required, the Grafana plugin catalog can be disabled using the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
```

File: docs/sources/administration/plugin-management/index.md [hunk 15]
```diff
@@ -80,91 +88,91 @@ The Plugin catalog allows you to browse and manage plugins from within Grafana.
 
 _Video shows the Plugin catalog in a previous version of Grafana._
 
-In order to be able to install / uninstall / update plugins using plugin catalog, you must enable it via the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
-Before following the steps below, make sure you are logged in as a Grafana administrator.
+In order to be able to install, uninstall, or update plugins using the plugin catalog, you must enable it with the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
```
Comment (sympatheticmoose) [inline docs/sources/administration/plugin-management/index.md:91]:
```suggestion
```

File: docs/sources/administration/plugin-management/index.md [hunk 16]
```diff
@@ -47,29 +53,31 @@ Apps can also add custom pages for things like control panels.
...
 
-With [RBAC]({{< relref "../roles-and-permissions/access-control/#about-rbac" >}}), it is now possible to customize access to app plugins.
+With [RBAC]({{< relref "../roles-and-permissions/access-control/#about-rbac" >}}), you can now customize access to app plugins.
 
-By default, Viewers, Editors and Admins have access to all App Plugins that their organization role allows them to access, thanks to the `fixed:plugins.app:reader` role.
+By default, the roles of Viewers, Editors and Admins have access to all app plugins that their Organization role allows them to access. Access is granted by the `fixed:plugins.app:reader` role.
 
 {{% admonition type="note" %}}
-Revoking this RBAC role from some users, will prevent them from accessing app plugins. But granting this RBAC role to users will only allow them to see app plugins their organization role allows them to see.
+Revoking the RBAC role from some users will prevent them from accessing app plugins. However, granting the RBAC role to users will only allow them to see app plugins their Organization role allows them to see.
```
[the remark this answers is not in the document]
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md]: I agree I re-read the NOTE and 

1st: I do not understand its ending where it says "allows them to see". It feels like some more information is missing. 

2nd: Just before the NOTE it explains the default behavior and then in note about revoking but not pointing to the doc link.

So yes this NOTE does not make that much sense here and can even confuse OSS users.

File: docs/sources/administration/plugin-management/index.md (hunk 15 above)
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md]: Agree on the changes due to the fact:

1. It is already enabled by default.
2. So, far in my experience while working with users, no one come and asked as how to disable this specific part.

File: docs/sources/administration/plugin-management/index.md [hunk 17]
```diff
@@ -15,11 +15,17 @@ weight: 600
...
 
-You can [install](#install-a-plugin) one of the plugins built by the Grafana community, or [build one yourself](/developers/plugin-tools).
+This guide shows you how to [install](#install-a-plugin) a wide range of the plugins that are built by Grafana Labs, commercial partners, our community, or plugins that you have [built yourself](/developers/plugin-tools).
 
-Grafana supports three types of plugins: [panels](/grafana/plugins?type=panel), [data sources](/plugins?type=datasource), and [apps](/grafana/plugins?type=app).
+## Types of plugins
+
+Grafana supports three types of plugins:
+
+- [Panels](/grafana/plugins/panel-plugins) - These plugins make it easy to create and add any kind of panel, to show your data, or improve your favorite dashboards.
```
[the remark this answers is not in the document]
Comment (usmangt) [reply docs/sources/administration/plugin-management/index.md:26]: @sympatheticmoose : How about:

- [Panels](/grafana/plugins/panel-plugins) - also known as visualizations, allows Grafana to use custom visualizations and display it in a Dashboard?

**Review by usmangt — COMMENTED** [review]
all ready to go live

**Comment by jdbaldry** [comment]
I updated the branch so that CI passes

**Comment by usmangt** [comment]
@sympatheticmoose or @josmperez can anyone of you please approve it? as we need one more to merge this PR into main

File: docs/sources/administration/plugin-management/index.md [hunk 18]
```diff
@@ -8,18 +8,25 @@ aliases:
...
-You can [install](#install-a-plugin) one of the plugins built by the Grafana community, or [build one yourself](/developers/plugin-tools).
+This guide shows you how to [install](#install-a-plugin) plugins that are built by Grafana Labs, commercial partners, our community, or plugins that you have [built yourself](/developers/plugin-tools).
 
-Grafana supports three types of plugins: [panels](/grafana/plugins?type=panel), [data sources](/plugins?type=datasource), and [apps](/grafana/plugins?type=app).
+## Types of plugins
+
+Grafana supports three types of plugins:
+
+- [Panels](/grafana/plugins/panel-plugins) - These plugins make it easy to create and add any kind of panel, to show your data, or improve your favorite dashboards.
+- [Data sources](/grafana/plugins/data-source-plugins) - These plugins allow you to pull data from various data sources such as databases, APIs, log files, etc and display it in the form of graphs, charts, and dashboards in Grafana.
```
Comment (josmperez) [inline docs/sources/administration/plugin-management/index.md:28]:
```suggestion
- [Data sources](/grafana/plugins/data-source-plugins) - These plugins allow you to pull data from various data sources such as databases, APIs, log files, and so on, and display it in the form of graphs, charts, and dashboards in Grafana.
```

File: docs/sources/administration/plugin-management/index.md [hunk 19]
```diff
@@ -80,91 +88,89 @@ The Plugin catalog allows you to browse and manage plugins from within Grafana.
...
-When the update is complete, you see a confirmation message that the uninstall was successful.
+When the update is complete, you'll see a confirmation message that the uninstall was successful.
 
 ## Install Grafana plugins
 
-Grafana supports data source, panel, and app plugins. Having panels as plugins makes it easy to create and add any kind of panel, to show your data, or improve your favorite dashboards. Apps enable the bundling of data sources, panels, dashboards, and Grafana pages into a cohesive experience.
+Grafana supports data source, panel, and app plugins.
 
-1. In a web browser, navigate to the official [Grafana Plugins page](/plugins) and find a plugin that you want to install.
+1. In a web browser, navigate to the [Grafana plugin catalog](/plugins) and find a plugin that you want to install.
```
Comment (josmperez) [inline docs/sources/administration/plugin-management/index.md:142]:
```suggestion
1. In a web browser, navigate to the [Grafana plugin catalog](https://grafana.com/plugins) and find a plugin that you want to install.
```

**Review by josmperez — APPROVED** [review]
Nice work! LGTM.

I just added a few minor suggestions (note that doc team's guidance for links has recently changed so that we now prefer full URLs such as https://grafana.com/plugins to /plugins even though both will work).
